src/app/shared/entry/launch-checker-workflow/launch-checker-workflow.component.html

```diff
@@ -0,0 +1,4 @@
+<div id="launchCheckerWorkflow" *ngIf="checkerWorkflowPath$ | async" class="well well-sm" tooltip="Commands for checking the checker workflow of this tool/workflow through Dockstore CLI">
```

Comment:
Hmmm, better to say that we run a checker workflow rather than check a check a checker workflow